en/user_guide/content.rst

```diff
@@ -0,0 +1,58 @@
+Contents
+========
+
+Contents are managed in the Back Office.
```

Comment:
I think it should be explained (or reminded) quickly that contents are created from content types, because if the specific page was not seen by the reader before it can be confusing.